google/cloud/pubsub/samples/subscription_admin_samples.cc

```diff
@@ -471,6 +471,26 @@ void AutoRun(std::vector<std::string> const& argv) {
   CreateSubscription(subscription_admin_client,
                      {project_id, topic_id, subscription_id});
 
+  // Any project can use another project's table, but the pubsub service account
+  // for the project needs to be granted access on the table. See
+  // https://cloud.google.com/pubsub/docs/create-bigquery-subscription#assign_bigquery_service_account
+  // for more details.
+  if (project_id == "cloud-cpp-testing-resources") {
```

Comment:
LMK if you think I should use env variables or there's a better way to do this

Reply:
We should avoid hard-coding the project name here.  Maybe use the `GOOGLE_CLOUD_PROJECT` environment variable or something?

Reply:
But then it won't run if you use another project-id, since the bigquery table does not exist.

Reply:
This is part of the `AutoRun` function, mostly used as part of the CI.  We have other cases where the project needs to have pre-existing resources and have them properly configured for the integration tests to run.

Reply:
Done